list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+$query = "SELECT * FROM books";
```

Comment:
Observations: No pagination or filtering on the list view.
Recommendations: Implement pagination and filtering to manage large datasets efficiently and improve user experience.